scubagoggles/reporter/reporter.py

```diff
@@ -144,6 +145,38 @@ def build_report_html(fragments : list, product : str,
     html = html.replace('{{TABLES}}', collected)
     return html
 
+def build_report_json(tenant_domain : str, report_stats: dict,
```

Comment:
The `Results` are currently structured like this:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/1151398b-71ef-4608-9f34-afbd6b1012e8)

But should be structured like this:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/ce6f6c91-c7d0-455b-ae5b-5e307f568fe6)